lib/osf-components/addon/components/contributor-list/contributor/template.hbs

```diff
@@ -1,4 +1,4 @@
-{{#if this.contributorLink~}}
+{{~#if this.contributorLink~}}
     {{!--
         using <a> instead of <Link> so handlebars `~` can eat up whitespace
         TODO: switch to `{{#osf-link}}` invocation when <Link> is renamed
```

Comment:
Minor: probably about time we did this. or delete the TODO?

Reply:
I think I'll leave that for another PR, but yeah, it's about time.